packages/web-runtime/src/components/UploadInfo.vue

```diff
@@ -167,13 +170,24 @@ export default defineComponent({
     runningUploads: 0, // all uploads (not files!) that are in progress currently
     bytesTotal: 0,
     bytesUploaded: 0,
+    uploadSpeed: 0,
     filesInEstimation: {},
     timeStarted: null,
     remainingTime: undefined
   }),
   computed: {
     ...mapGetters(['configuration']),
 
+    uploadDetails() {
+      if (this.uploadSpeed === 0) {
+        return ''
+      }
+      const todo = filesize(this.bytesUploaded)
+      const done = filesize(this.bytesTotal)
```

Comment:
Naming of these two variables seems off

Reply:
Updated, thx! (no clue what I was thinking 😶‍🌫️ )